pkgs/native_assets_builder/lib/src/model/kernel_assets.dart

```diff
@@ -6,7 +6,7 @@
 /// kernel file.
 ///
 /// The `native_assets.yaml` embedded in a kernel file has a different format
```

Comment:
`native_assets.yaml`?

Reply:
yep, that's not part of the protocol it's internal to the native_assets_builder and the CFE.

Maybe we should consider moving that to JSON as well. If so, in a separate PR.